playbooks/portals-setup-following.yml

```diff
@@ -27,7 +27,7 @@
           become: True
     
     - name: Include firewall setup
-      include_tasks: tasks/portal-firewall-ufw-setup.yml
```

Comment:
We are replacing this one, should we also replace the one in `portal-setup-ufw.yml` playbook, or that one is fine?

Reply:
In `playbooks/portals-setup-following.yml`
we call `tasks/portal-firewall-setup.yml`
which calls both:
- `tasks/portal-firewall-iptables-block-unblock.yml`
- `tasks/portal-firewall-ufw-setup.yml`

We have 2 now firewall playbooks:
- `playbooks/portals-block-unblock-incoming-traffic.yml` - for un-/blocking incoming traffic in iptables
- `playbooks/portal-setup-ufw.yml` - for the rest of firewall done via ufw
so we can target both individually.